tests_new/integration_tests/README.md

```diff
@@ -3,7 +3,18 @@ The purpose of these tests is to automatically validate functionalities of data.
 
 ## Pre-requisites
 - A real deployment of data.all in AWS
-- 4 Cognito users (at the moment only Cognito is supported) like the ones defined in `conftest`(e.g. `testUser1` with password `Pass1Word!`)
+- A SecretManager secret (`'{self.resource_prefix}-{target_env["envname"]}-cognito-test-users'`) with the followings contents
+    ```
```

Comment:
I know, that's for test purposes, but why expose it in code (especially opensource)? May be it's better just to leave comment like: `userdata with username, password and groups`.

Reply:
I just gave an example password, real deployments should use a different one. Is your concern that people will just copy and paste the example and hence make themselves vulnerable?